Makefile

```diff
@@ -27,7 +27,7 @@ SILENT ?= @
 UNIT_TEST_IGNORE := "stackrox/rox/sensor/tests|stackrox/rox/operator/tests|stackrox/rox/central/reports/config/store/postgres|stackrox/rox/central/complianceoperator/v2/scanconfigurations/store/postgres|stackrox/rox/central/auth/store/postgres|stackrox/rox/scanner/e2etests"
 
 ifeq ($(TAG),)
-TAG=$(shell git describe --tags --abbrev=10 --dirty --long --exclude '*-nightly-*')$(MAIN_TAG_SUFFIX)
```

Comment:
One thing I'd also ask is to adjust operator's Makefile which does not need to handle cases like `4.5.x` and `3.0.62.1`.
https://github.com/stackrox/stackrox/blob/a8df2c257610b1f3563dbc053a900b7070358063/operator/Makefile#L6-L8

It can simply become
```Makefile
VERSION ?= $(shell $(MAKE) --quiet --no-print-directory -C .. tag)
```
since the Makfile in repo root now does handling of `.x` and versions like `3.0.y.z` are long gone.